src/services/api/apiServices.tsx

```diff
@@ -7,11 +7,12 @@
 //         'Content-Type': 'application/json; charset=UTF-8',
 //     },
 // });
-import {configData} from '@/utils/config';
-import axios, {AxiosInstance} from 'axios';
+import { configData } from '@/utils/config';
+import axios, { AxiosInstance } from 'axios';
 
 const apiServices: AxiosInstance = axios.create({
-  baseURL: `https://dev-gm.vercel.app/api/v1/`,
+  // baseURL: `https://dev-gm.vercel.app/api/v1/`,
+  baseURL: `http://192.168.1.56:5000/api/v1/`,
```

Comment:
Make sure to use URL from ENV file not hard coded and this is your local URL not dev url please pick from env and that to dev-gm